records_mover/db/postgres/sqlalchemy_postgres_copy.pyi

```diff
@@ -0,0 +1,2 @@
+def copy_from(source, dest, engine_or_conn, columns=(), **flags):
+    ...
```

Comment:
I was annoyed at mypy which totally killed my coverage metrics when I brought in the vendored file.  Thankfully it turns out you can drop a `.pyi` file in and add types there.  Since there's only one function referenced by everything else, it also turned out just dropping that here made it stop paying attention to the type coverage of the full file (including all of the helper functions inside) and just focus on this function.